packages/web-components/fast-components/src/skeleton/index.ts

```diff
@@ -0,0 +1,12 @@
+import { customElement } from "@microsoft/fast-element";
+import { Skeleton, SkeletonTemplate as template } from "@microsoft/fast-foundation";
+import { SkeletonStyles as styles } from "./skeleton.styles";
+
+@customElement({
+  name: "fluent-skeleton",
```

Comment:
Shouldn't it be "fast-skeleton" in this directory?

Reply:
Yeah, this will get pulled and moved to Fluent once we've had a solid review of it. I had @litteredwitherrors move this out of Draft because it wasn't getting any looks 😄